__tests__/app.test.js

```diff
@@ -124,6 +124,98 @@ describe("Articles", () => {
+        body: "some gifs",
+        created_at: "2020-11-03T09:12:00.000Z",
+        votes: 0,
+        comment_count: 2,
+      };
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body: { article } }) => {
+          expect(article[0]).toMatchObject(articleExample);
+        });
+    });
+    test("sortby sorts any valid column by default desc", () => {
+      const articleExample = {
+        title: "Student SUES Mitch!",
+        topic: "mitch",
+        author: "rogersop",
+        body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
+        created_at: "2020-05-06T01:14:00.000Z",
+        votes: 0,
+      };
+      return request(app)
+        .get("/api/articles?sort_by=author")
+        .expect(200)
+        .then(({ body: { article } }) => {
+          expect(article[0]).toMatchObject(articleExample);
+        });
+    });
+    test("status: 404 when given a non-existent sortby", () => {
+      return request(app)
+        .get("/api/articles?sort_by=writer")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Query does not exist");
+        });
+    });
+    test("orderby orders by default desc but can also asc", () => {
+      const articleExample = {
+        article_id: 6,
+        title: "A",
+        topic: "mitch",
+        author: "icellusedkars",
+        body: "Delicious tin of cat food",
+        created_at: "2020-10-18T01:00:00.000Z",
+        votes: 0,
+        comment_count: 1,
+      };
+      return request(app)
+        .get("/api/articles?sort_by=title&order_by=ASC")
+        .expect(200)
+        .then(({ body: { article } }) => {
+          expect(article[0]).toMatchObject(articleExample);
+        });
+    });
+    test("status: 404 when given a non-existent orderby", () => {
+      return request(app)
+        .get("/api/articles?order_by=up")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Query does not exist");
+        });
+    });
+    test("topic filters articles by topic value in query", () => {
+      const articleExample = {
+        title: "UNCOVERED: catspiracy to bring down democracy",
+        topic: "cats",
+        author: "rogersop",
+        body: "Bastet walks amongst us, and the cats are taking arms!",
+        created_at: expect.any(String),
+        votes: 0,
+      };
+      return request(app)
+        .get("/api/articles?topic=cats")
+        .expect(200)
+        .then(({ body: { article } }) => {
+          expect(article[0]).toMatchObject(articleExample);
+        });
+    });
+    test("status: 404 when given a non-existent topic", () => {
+      return request(app)
+        .get("/api/articles?topic=none")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Query does not exist");
+        });
+    });
```

Comment:
Please add a test for when a query is a non-existent query (ie. `/api/articles?notValid=nope`), this should not throw an error but should return with the default queries set.